### DISCUSSION TITLE
Quake mode ignoring "initialPosition", "initialCols","initialRows" settings

### DISCUSSION THREAD

**Discussion by mkanet** [microsoft/terminal#14926] [Q&A] [answered]
I can't seem to get quake mode to open in the position and size specified by MS Terminal settings: `initialPosition`, `initialCols`,`initialRows`.  Quake mode is taking up the entire width of my large display; completely ignoring these settings.

Is this a bug or am I doing something wrong?  I couldn't find any information on this when searching online.

**Comment by j4james** [comment, marked as answer]
I believe this is intentional. It's emulating the way the drop-down console worked in the [Quake video game](https://en.wikipedia.org/wiki/Quake_(video_game)). Quoting from the [Terminal documentation](https://learn.microsoft.com/en-us/windows/terminal/tips-and-tricks#quake-mode):
> When a window is in quake mode:
> * The terminal is automatically snapped to the top half of the monitor.
> * The window can no longer be resized horizontally or from the top. It can only be resized on the bottom.

**Reply by mkanet** [reply]
Thank you, I was hoping there was a way to do this natively.  I guess I'll have to rely on https://github.com/flyingpie/windows-terminal-quake; which makes quake mode much more flexible.

**Comment by zadjii-msft** [comment]
You can always just use the [`globalSummon`](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#global-summon) action (rather than the `quakeMode` action), which is more flexible. That can be used to hotkey-activate _any_ terminal window you have open, not just the one named "_quake" (which has special properties)
